service.md

```diff
@@ -1,10 +1,60 @@
 ### Expose VirtualMachine as a service
 
-Once the VirtualMachine is started, in order to connect to a VirtualMachine, you can create a `Service` object for a VirtualMachine.
+Once the VirtualMachine is started, in order to connect to a VirtualMachine, you can create a `Service` object for a VirtualMachine. Currently, three types of service are supported: `ClusterIP`, `NodePort` and `LoadBalancer`. The default type is `ClusterIP`.
 
-### Expose VirtualMachine as a NodePort service                                                                                                                                               
+### Expose VirtualMachine as a ClusterIP service
```

Comment:
@rmohr @fabiand Just a thought.

For our user documentation, I think we should be talking more about k8s services in the context of OfflineVirtualMachines (OVMs) specifically. That's one place where the value is very apparent.

OVMs are meant to be persistent and immortal, but the underlying tech the supports the OVMs (pods) are mortal. This means that every time we start/stop/start an OVM, that specific vm instance gets a new IP address.

OVMs coupled with a Service endpoint means users can reach the OVM using a static endpoint that will remain consistent throughout OVM restarts.

Reply:
https://github.com/kubevirt/kubevirt/issues/861